### PR TITLE
RV ASMFT 344

### PR DESCRIPTION
**Pull request by robertvazquezlegalshield**
This is for the successful login to the Partner app and also to add the new secure login credentials created just for the Partner app.

### CODE REVIEW

File: demo/adonis/partner/login_successful.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+var i = require('../../tools.js')
+var secure = require('../../secret.js')
+var ENTER = '\uE007'
```
Comment (TrentonWalkup) [inline demo/adonis/partner/login_successful.js]: You don't need this.

File: demo/adonis/partner/login_successful.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+who('As a Partner I can', function() {
+where('https://partner.sandbox.legalshield.com')
+what('login successfully ASMFT-343', function () {
+  click('mat-form-field.mat-form-field:nth-child(2) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1)')
+    enterText('#mat-input-0', secure.partner_login)
+    enterText('#mat-input-1', secure.partner_password)
+    click('.login-button')
+    let welcomeMessage = browser.isVisible('.search-empty > h4:nth-child(2)')
```
Comment (TrentonWalkup) [inline demo/adonis/partner/login_successful.js]: very nice

File: demo/secret.js [hunk 3]
```diff
@@ -5,6 +5,8 @@ var office_365_email = process.env.OFFICE_365_EMAIL
 var office_365_password = process.env.OFFICE_365_PASSWORD
 var lsatwork_username = process.env.LSATWORK_USERNAME
 var lsatwork_password = process.env.LSATWORK_PASSWORD
+var partner_login = process.env.PARTNER_LOGIN
```
Comment (TrentonWalkup) [inline demo/secret.js]: please share creds with qa privately to put in .env files.

**Comment by Bradern8t0r** [comment]
LGTM

**Pull request merged**